app/src/main/java/org/thunderdog/challegram/telegram/Tdlib.java

```diff
@@ -518,7 +518,7 @@ public TdlibCounter getCounter (@NonNull TdApi.ChatList chatList) {
     this.isLoggingOut = account.isLoggingOut();
 
     this.parameters = new TdApi.TdlibParameters(
-      false, null, null, // updateParameters
```

Comment:
This, like other similar changes in this PR, shouldn't been a part of the submission. Kindly exclude these changes from this PR.